scripts/clean_osm_data.py

```diff
@@ -852,7 +899,7 @@ def clean_data(
 
     if os.path.getsize(input_files["lines"]) > 0:
         # Load raw data lines
-        df_lines = gpd.read_file(input_files["lines"])
+        df_lines = load_network_data("lines")
```

Comment:
since each load_network_data shall include a parameter "clean_osm_data_options", this needs to be added.
This argument needs to be added to the function clean_data as well.
Note also that parameters are now passed using "snakemake.params" instead of "snakemake.config", so 
`clean_osm_data_options=snakemake.params["clean_osm_data_options"]`

Cool! :D
It would be nice to externalize the custom_data_paths, but in this PR we can keep this structure :)

Reply:
Hey @davide-f ! thanks for the comments, I think these have been taken care of with some considerations:
*For spaces in the code, extra empty lines or separations were deleted (I tried to match the format of other functions)
*Regarding the additional parameter in the load_network_data function, this has been added as "data_options" (I used that name given that when I used "clean_osm_data_options" I got some naming convention errors)
*All sections where the load_network_data function is used have been updated to include the new argument (data options, which is defined at the end of the code with the rest of the variables)
*The data_options parameter has been added in the clean_data function as well, and the argument was passed as suggested (data_options=snakemake.params["clean_osm_data_options"]).
*A line has been added in the release_notes explaining the function but it seems I don't have writing access to change it (to keep it consistent with the latest version I updated the file before adding the note)

I hope that all of this makes sense and that I understood everything correctly, but again, let me know if something else needs to be done.